src/pages/map/map.spec.ts

```diff
@@ -277,12 +271,7 @@ describe('MapPage', function () {
       }));
 
       it('should center the map without changing the zoom level', fakeAsync(function () {
-
-        map.setZoom(10);
-
-        // setZoom calls setView
-        expect(setViewSpy.args[1]).to.eql([L.latLng(46.183541, 6.100234), 10, { zoom: undefined }]);
-        expect(setViewSpy).to.have.been.calledTwice;
+        setUpMap({ zoom: 10 });
```

Comment:
Why is this solution more relevant than using `beforeEach` and setting the zoom manually in this particular test ?
Seems to me that it defeats the purpose of using `beforeEach`

Reply:
Because `beforeEach` is common to all tests and you have no way to change what it does for a particular test. In this case, I need to set up the map with a different initial zoom level in the test "should center the map without changing the zoom level":

```
setUpMap({ zoom: 10 });
```

Reply:
But... it was done here : https://github.com/MediaComem/biopocket-mobile/pull/7/files/3af48366de4b58f5770d7b3f6cddf281815e8cf6#diff-7ccad74438419eb4539230a1fd447f5cL281